profiles/base.yaml

```diff
@@ -121,6 +121,7 @@ custom_loot_type:
 last_rites: false
 ambush_location: back
 warhorn:
+prayer_mat:
```

Comment:
What do you think about `combat_trainer_prayer_mat`?

Reply:
Counteroffer: what I *actually* want is to use this setting for both theurgy.lic and for combat-trainer.lic .  The problem is that would screw up anyone who has already adopted `theurgy_prayer_mat` (but the only side effect would be that they'd pray at their local altar).  I'd like to make a `theurgy_use_prayer_mat` setting that just uses `prayer_mat` instead -- as you noted, no one has two prayer mats. OK to get rid of the old setting and sub in a new one?

Reply:
I don't have the code in front of me, but couldn't the player turn off the commune in theurgy if they only wanted to use their prayer mat in combat-trainer?

Reply:
Unfortunately I set it up so that the presence of `theurgy_prayer_mat` causes it to be used.  And it's several rituals that get subbed out.

Reply:
Got it.

To back up a step then, I'm not sure I understand your counteroffer. Are you suggesting adding `theurgy_use_prayer_mat` as a boolean which is used in theurgy and determines whether or not to use the prayer mat?

Reply:
That's right; I've gone ahead and done this.  Take a look and see if you think the new setup is reasonable.